bedrock/legal/templates/legal/eula.html

```diff
@@ -32,11 +32,12 @@
   </section>
 
   <section class="archive">
-    <h3>Outdated policies</h3>
-    <p>Mozilla Firefox End-user Software License Agreement:</p>
```

Comment:
Instead of deleting this string could we just omit `Firefox` and change it to `Mozilla End-user Software License Agreement:`?